ff-macros/src/lib.rs

```diff
@@ -0,0 +1,88 @@
+#![deny(
+    warnings,
+    unused,
+    future_incompatible,
+    nonstandard_style,
+    rust_2018_idioms
+)]
+#![forbid(unsafe_code)]
```

Comment:
I'm confused as to why we want forbid here vs deny

Reply:
forbid cannot be bypassed with an `allow`

Reply:
Ohh, that makes sense. Why don't we default to having things be forbidden, unless we want a local allow then?

Reply:
I think in most crates we have a forbid?